### DISCUSSION TITLE
To Splash, or not to Splash, that is the question..?

### DISCUSSION THREAD

**Discussion by ntoll** [mu-editor/mu#1873] [General]
We need to agree on the the behaviour of the splash screen. Should it be on top or not? Etc... Opinions please...!

**Comment by ZanderBrown** [comment]
How are we defining "on top"? It's worth noting that "always on top" isn't available to clients in a modern linux world

**Reply by carlosperate** [reply]
Yeah, that's good to clarify, as this is an issue in Windows only (as far as a I know).

**Comment by carlosperate** [comment]
So as far as I know these are the issues we are facing:

- With "splash on top" on Windows on a first run (that can take several minutes) the splash screen can take a large portion of the screen and basically blocks you from using the PC until it's done
- Without it, on Windows, the splash screen can be lost behind other programme windows and it's hard to tell that is still running.
    - Questions:
        - Is there still an icon in the tasks bar?
        - Can the splash screen still be shown with alt+tab?
